com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -580,6 +580,20 @@ private void OnEnable()
         /// <param name="prefab"></param>
         public void RemoveNetworkPrefab(GameObject prefab) => PrefabHandler.RemoveNetworkPrefab(prefab);
 
+        /// <summary>
+        /// Sets the maximum size of a single non-fragmented message (or message batch) passed through the transport.
+        /// This should represent the transport's MTU size, minus any transport-level overhead.
+        /// </summary>
+        /// <param name="size"></param>
+        public void SetMaxSingleMessageSize(int size) => MessageManager.SetMaxSingleMessageSize(size);
```

Comment:
should we name this simply MTU (`SetMaximumTransmissionUnit`) for more obvious and transparent communication through APIs?